arrMethodsLab.js

```diff
@@ -0,0 +1,409 @@
+// # Exercises
```

Comment:
Overall good job , you did not create function for your answers. Please make sure you are creating functions for your answers. console.log() is a function used to print information to the console. return on the other hand is a call to pass some value back up to where the call was made.